README.md

```diff
@@ -4,25 +4,20 @@ Savon Multipart [![Build Status](https://secure.travis-ci.org/savonrb/savon-mult
 Adds multipart support (SOAP with Attachments) to [Savon](https://github.com/savonrb/savon).  
 Please test and provide feedback so we can support as many multipart-soap messages as possible.
 
+Installation
+------------
 
-Savon2
-------
-
-Recently Savon launched a rewrite that broke compatibility with savon-multipart. If you're using savon
-version 1, this should just work, but if you'd like to use Savon2, you will probably want to specify
-something like this in your Gemfile
-
+Savon Multipart is available through [Rubygems](http://rubygems.org/gems/savon-multipart) and can be installed via:
 
 ```
-gem 'savon-multipart',       :git => 'https://github.com/tjarratt/savon-multipart.git', :branch => 'savon2'
+$ gem install savon-multipart
 ```
 
-
-Installation
+Usage
 ------------
 
-Savon Multipart is available through [Rubygems](http://rubygems.org/gems/savon-multipart) and can be installed via:
+Require the gem once 'savo' is loaded:
```

Comment:
There's a type `savo` => `savon`. Savon is automatically required in `savon-multipart.rb` and you don't have to load it manually, it's automatic. I think it's safe to remove this line to avoid confusion.